elixir/action.yml

```diff
@@ -0,0 +1,26 @@
+name: "Snyk Elixir"
+description: "Check your Elixir application for vulnerabilities using Snyk"
+author: "Snyk Ltd"
+branding:
+  icon: "alert-triangle"
+  color: "yellow"
+inputs:
+  command:
+    description: "Which Snyk command to run, defaults to test"
+    default: test
+  args:
+    description: "Additional arguments to pass to Snyk"
+  json:
+    description: "Output a snyk.json file with results if running the test command"
+    default: false
+runs:
+  using: "docker"
+  image: "docker://snyk/snyk:elixir"
```

Comment:
question: do we want to consider pinning a version? It would require adding an updated version in snyk images, as the only pinned version is the rather old [elixir 1.12](https://hub.docker.com/r/snyk/snyk/tags?name=elixir)